__tests__/app.test.js

```diff
@@ -31,31 +31,91 @@ describe('app routes', () => {
       return client.end(done);
     });
 
-    test('returns animals', async() => {
+    test('returns todos', async() => {
```

Comment:
This test is grabbing all the user and their todos. You may want one where you grab a single user but all of their todos.